### ISSUE TITLE
WARNING: key 'sql_query_info' was permanently removed from Sphinx configuration

### ISSUE THREAD

**Issue by joker-777** [pat/thinking-sphinx#882]
Hi guys,

we get the following warnings when we do a rebuild after we upgraded sphinx to 2.2.6.
```
WARNING: key 'sql_query_info' was permanently removed from Sphinx configuration. Refer to documentation for details.
WARNING: key 'charset_type' was permanently removed from Sphinx configuration. Refer to documentation for details.
WARNING: key 'enable_star' was permanently removed from Sphinx configuration. Refer to documentation for details.
WARNING: key 'sql_query_info' was permanently removed from Sphinx configuration. Refer to documentation for details.
WARNING: key 'charset_type' was permanently removed from Sphinx configuration. Refer to documentation for details.
```
as far as we can see sphinx still behaves properly  but we were wondering if you aware of it and when you are going to fix it.

**Comment by pat** [comment]
Thanks for bringing this up - I've just pushed commits to the develop branch which remove both sql_query_info and charset_type. It'll be part of the next release (not yet sure if that'll be 3.2 or 3.1.4).

enable_star will be coming from your config/thinking_sphinx.yml or specific index definitions though :)

**Comment by joker-777** [comment]
Hi pat, thanks of taking care of. I tested it and the warnings are gone now. The star is now enabled by default, right? Thanks for the your great work.f

**Comment by pat** [comment]
That's correct, enable_star is considered always-enabled for Sphinx 2.2.2 onwards - and thanks for the confirmation and kind words :)

**Comment by jogaco** [comment]
I'm having sql_query_info and charset_type warnings. 
My versions: Sphinx 2.2.7, TS 3.1.3

**Comment by pat** [comment]
Just released 3.1.4 which includes these fixes :)
